controller/scheduler/mesos_marathon.py

```diff
@@ -0,0 +1,111 @@
+import re
+import time
+from django.conf import settings
+from marathon import MarathonClient
+from marathon.models import MarathonApp
+from .states import JobState
+from docker import Client
+from .fleet import FleetHTTPClient
```

Comment:
These imports should have a blank line between each group as per [PEP8](https://www.python.org/dev/peps/pep-0008/#imports):

> Imports should be grouped in the following order:
> - standard library imports
> - related third party imports
> - local application/library specific imports
>
> *You should put a blank line between each group of imports.*